packages/explorable-explanations/src/topics/index.ts

```
@@ -187,10 +187,14 @@ export function topicsAnimation(
       app.lastFrameCount = 0;
       app.prevVisitedCircleIndex = -1;
       app.inspectedCircles.clear();
-      p?.clear();
-      p.background(255);
-      p.pixelDensity(2);
-      app.drawTimeline(config.timeline.position, epoch);
+      try {
+        p.clear();
+        p.background(255);
+        p.pixelDensity(2);
+        app.drawTimeline(config.timeline.position, epoch);
+      } catch (error) {
+        // fails silently if canvas was removed
+      }
```

What was the error here?

---

This was a very weird error and I am not 100% sure how it happens, where at some point because `p` gets removed from memory and it trigger an error. I didn't want to spend more time figuring it out because I already refactor all this code in another PR and if we ignore the error it doesn't afect the animation at all.